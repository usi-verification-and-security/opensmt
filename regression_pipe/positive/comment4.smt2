(set-logic QF_LRA)(declare-fun X () Real)(assert (< 1 X)) (check-sat) (get-value (X)); comment((asse
(check-sat) ; Another comment
(check-sat)
(exit)
