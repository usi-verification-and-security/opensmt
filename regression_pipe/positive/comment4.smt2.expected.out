sat
((X 2))
sat
sat
